validator/validators/performance/performance_validator.go

```diff
@@ -45,19 +46,34 @@ func NewPerformanceValidator(vConfig models.ValidateConfig) models.ValidatorFact
 }
 
 func (s *PerformanceValidator) CheckData(startTime, endTime time.Time) error {
-	metrics, err := s.GetPerformanceMetrics(startTime, endTime)
-	if err != nil {
-		return err
-	}
+	if strings.Contains(s.vConfig.GetTestCase(), "windows") {
+		stat, err := s.GetWindowsPerformanceMetrics(startTime, endTime)
```

Comment:
nit: Instead of having specific OS specific methods, might be cleaner to make these interface methods and have linux & windows implement each.

Reply:
This may require refactor for Linux tests. We should do this in a separate PR.